db/schema.rb

```diff
@@ -381,15 +382,21 @@
     t.string "campus_type"
     t.string "parent_facility_code_id"
     t.bigint "version_id"
+    t.index "lower((address_1)::text) gin_trgm_ops", name: "index_institutions_on_address_1_trgm", using: :gin
+    t.index "lower((address_2)::text) gin_trgm_ops", name: "index_institutions_on_address_2_trgm", using: :gin
+    t.index "lower((address_3)::text) gin_trgm_ops", name: "index_institutions_on_address_3_trgm", using: :gin
     t.index "lower((institution)::text) text_pattern_ops", name: "index_institutions_institution_lprefix"
     t.index ["address_1"], name: "index_institutions_on_address_1"
     t.index ["address_2"], name: "index_institutions_on_address_2"
     t.index ["address_3"], name: "index_institutions_on_address_3"
     t.index ["city"], name: "index_institutions_on_city"
```

Comment:
Should we drop these other indexes?

Reply:
Yes, I was planning on dropping them in a follow up migration since I think we'd want to merge that after https://github.com/department-of-veterans-affairs/gibct-data-service/pull/553 (unless you think it'd be okay to do it in one PR)

Reply:
Should be fine to do in one PR.  You're essentially replacing them.  Keeping them around will just slow down inserts into `institutions`.  And dropping indexes is quick. Not a huge deal either way.